clj-refactor.el

```diff
@@ -155,6 +155,14 @@ as can be."
   :group 'cljr
   :type 'boolean)
 
+(defcustom cljr-suppress-outside-project-warning nil
+  "If t, no warning is printed when starting a REPL outside a project.
+By default, a warning is printing in this case since clj-refactor
+will not work as expected in such REPLs and therefore the
```

Comment:
Cut everything after "and therefore", as that last part only confuses things.

Reply:
Done.